### PR TITLE
[5.2] Customize error message for ThrottleRequests

### PR DESCRIPTION
**Pull request by Davitig**

### CODE REVIEW

File: src/Illuminate/Routing/Middleware/ThrottleRequests.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace Illuminate\Routing\Middleware;
 
+use Lang;
```
Comment (GrahamCampbell) [inline src/Illuminate/Routing/Middleware/ThrottleRequests.php]: We can't assume this is alised.

**Comment by GrahamCampbell** [comment]
This introduces coupling on our routing component to the language component.

**Comment by TheGIBSON** [comment]
Then what will be the optimal solution?

**Comment by taylorotwell** [comment]
Not customizable at the moment.

**Pull request closed without merging**